ArticleTemplates/articleTemplateContainer.html

```diff
@@ -43,9 +43,8 @@ <h1 class="headline selectable" id="headline">__TITLE__ __REVIEW_RATING__</h1>
         </div>
 
         <div class="tags" id="tags">
-            <ul class="inline-list" id="tag-list">
-                <li class="inline-list__item screen-readable">Tags:</li>
-            </ul>
+            <span class="screen-readable">Tags:</span>
```

Comment:
Didn't see a reason why this couldn't be span, let me know if there is a reason!